apps/api/src/app/notification-template/dto/update-notification-template-request.dto.ts

```diff
@@ -37,9 +37,6 @@ export class UpdateNotificationTemplateRequestDto implements ICreateNotification
   @IsMongoId()
   notificationGroupId: string;
 
-  @ApiPropertyOptional()
-  active?: boolean;
```

Comment:
this property from DTO is not even passed to the usecase in controller.

Application has seperated endpoint to update status